easybuild/easyblocks/c/clang_aomp.py

```diff
@@ -230,7 +232,7 @@ def _configure_llvm(self, component):
         component['configopts'] = ' '.join([
             "-DLLVM_ENABLE_PROJECTS='clang;lld;compiler-rt'",
             "-DCLANG_DEFAULT_LINKER=lld",
-            "-DGCC_INSTALL_PREFIX=$EBROOTGCC",
+            "-DGCC_INSTALL_PREFIX=$EBROOTGCCCORE",
```

Comment:
@akesandgren Don't we want to consider both?
How about using a construct like: `${EBROOTGCC:-${EBROOTGCCCORE}}`, to use `$EBROOTGCCCORE as a fallback if `$EBROOTGCC` is not defined? (or other way around, not sure what is better)

Reply:
Ahh, forgot about that one...
${EBROOTGCC:-${EBROOTGCCCORE}} is probably better, or pick up on it in python code first with getenv(...)

Reply:
If `${EBROOTGCC}` is set, it should also point to the installation prefix of the actual compiler, because we use `altroot = 'GCCcore'` in the `GCC` easyconfigs.

Picking up the value with `os.getenv` is probably better, since it makes the command being run more explicit (you don't need to go and figure out what `$EBROOTGCCCORE` is set to when that command is beingeun

Reply:
Since Clang-AOMP is at GCCcore level picking up os.getenv('EBROOTGCCCORE') should be enough, but should we protect against someone trying to use this on some completely different base toolchain?

Reply:
@boegel any more thoughts on this before I change this to use os.getenv instead of $EBROOT...

Reply:
@boegel ping

Reply:
@boegel ping again

Reply:
@akesandgren Yes, we should, we should consider both `$EBROOTGCC` and `$EBROOTGCCCORE`, as suggested.

If we want to make this backwards compatible, we should consider `$EBROOTGCC` first, and only pick up on `$EBROOTGCCCORE` if `$EBROOTGCC` is not defined.